tests/e2e/actions.ts

```diff
@@ -38,7 +38,19 @@ const Actions = () => {
     await element.multiTap(times);
   };
 
-  return { waitForElement, typeText, tap, multiTap };
+  const scrollToElement = async (
+    element: Detox.DetoxAny,
+    scrollViewID: string,
+    NumberOfPixelsToScroll: number,
+    direction: Detox.Direction,
+  ) => {
```

Comment:
I like it, but a few things:
- I'd suggest renaming `scrollViewID` to `scrollableElement` and instead of `string` type I'd make it `Detox.Matcher`. This will make the function more flexible, I believe.
- before invoking `waitFor()` you should wait until the scrollable element is visible. So, insert `this. waitForElement()` before.
- In JS there's a common practice to put all options, like `NumberOfPixelsToScroll` (btw. it should be camel case, not pascal one) or `direction` into `options` object. So, it'd be like that: 
``` 
const scrollToElement = async (element: Detox.DetoxAny, scrollViewId: string, options: ScrollToElementOptions) => {}
```
- I think options like `NumberOfPixelsToScroll` and `direction` should be optional - it most cases we want to scroll down and at the same speed 🙃

Reply:
I'm not sure if in this case direction and pixels numbers can be optional:
![image](https://user-images.githubusercontent.com/72853287/104593777-7e497000-5670-11eb-87cd-abfe520ce620.png)
Link to full doc: https://github.com/wix/Detox/blob/master/docs/APIRef.ActionsOnElement.md

Reply:
They can't be optional for method `whileElement` but can be optional for `scrollToElement()` - `scrollToElement()` should simply have default values for these two options

Reply:
Done